packages/pyright-internal/src/tests/languageServer.test.ts

```diff
@@ -235,4 +239,153 @@ describe(`Basic language server tests`, () => {
         // Make sure the error has a special rule
         assert.equal(diagnostic.diagnostics[0].code, 'reportUnknownParameterType');
     });
+    describe('module renaming', () => {
```

Comment:
`foo/bar/baz.py` -> `bat.py`
```py
# foo/bar/qux.py
from . import baz
```
et al

Reply:
pylance doesn't support this so i'm not gonna bother for now. it can be done in a follow up PR